src/config.ts

```diff
@@ -12,6 +13,28 @@ const AWSX_PROFILE_PATH = path.join(AWSX_HOME, 'profiles');
 const AWS_CREDENTIALS_PATH = path.join(AWS_HOME, 'credentials');
 const AWS_CONFIG_PATH = path.join(AWS_HOME, 'config');
 
+const fileCheck = (): void => {
+  try {
+    const missingFiles: string[] = [];
+
+    !fs.existsSync(AWSX_PROFILE_PATH) && missingFiles.push(AWSX_PROFILE_PATH);
```

Comment:
When someone runs awsx for the first time they will not have this file which will throw an error and prevent them from setting up a profile.